climpred/bootstrap.py

```diff
@@ -51,6 +51,44 @@ def _resample(hind, resample_dim):
     return smp_hind
 
 
+def _resample_idx(init, bootstrap, dim='member', replace=True):
+    """Resample bootstrap times over dim."""
+
+    def xr_broadcast(x, idx):
+        return np.moveaxis(x.squeeze()[idx.squeeze().transpose()], 0, -1)
+
+    # I get errors if I have a 0 in dim
```

Comment:
Can you post an example of this? I can't replicate any errors.

The following works for me and includes a 0 in the member dimension.

```python
init = np.arange(1990, 2000)
member = np.arange(0, 10)
lead = np.arange(3) + 1

da = xr.DataArray(np.random.rand(len(init), len(member), len(lead)),
                  dims=['init', 'member', 'lead'],
                  coords=[init, member, lead])

_resample_idx(da, 10)
```

Reply:
this was somewhere in my mockup phase. cannot reproduce. therefore delete.